### PR TITLE
Remove all Plone 4 artifacts from the #1119 PR

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]
Man. How I hate those if statements that indent 90% of the code...

**Comment by tisto** [comment]
@sneridagh I think we should have a changelog entry when we clean up things. In case of a clean up break something in the future it might be very helpful to being able to check the changelog. I'd use the bugfix section in the changelog for this because it should be a non-breaking change.

File: src/plone/restapi/services/configure.zcml [hunk 1]
```diff
@@ -36,10 +36,7 @@
   <include package=".users"/>
   <include package=".vocabularies"/>
   <include package=".workflow"/>
-  <include
-      package=".workingcopy"
-      zcml:condition="have plone-5"
-      />
+  <include package=".workingcopy"/>
   <include
```
Comment (thet) [inline src/plone/restapi/services/configure.zcml:40]: all other ``zcml:condition="have plone-5"`` could be removed. unless this exceeds the scope of this PR, which I don't know.

**Comment by davisagli** [comment]
@sneridagh Is there something here which is still relevant, or should we just close this old PR?

**Comment by sneridagh** [comment]
@davisagli don't recall what happened to this PR but it seems all borked out. Let's better close it. Don't remember if we already removed the Plone 4 artifacts somewhere else.

**Pull request closed without merging**